src/rgw/rgw_file.h

```diff
@@ -510,7 +516,12 @@ namespace rgw {
     int readdir(rgw_readdir_cb rcb, void *cb_arg, uint64_t *offset, bool *eof,
 		uint32_t flags);
     int write(uint64_t off, size_t len, size_t *nbytes, void *buffer);
-    int write_finish();
+
+    int commit(uint64_t offset, uint64_t length, uint32_t flags) {
+      return 0;
+    }
```

Comment:
yeah, atm, I'm just passing this result up through ganesha, which would have the result adam describes;  we -could- return ENOTSUP here if we strongly prefer it, but then the rgw fsal would need to translate it to this